server/chat-plugins/datasearch.ts

```diff
@@ -1613,7 +1613,7 @@ function runMovesearch(target: string, cmd: string, canAll: boolean, message: st
 
 	const getFullLearnsetOfPokemon = (species: Species) => {
 		let usedSpecies: Species = Utils.deepClone(species);
-		let usedSpeciesLearnset: LearnsetData = Utils.deepClone(mod.species.getLearnset(usedSpecies.id));
+		let usedSpeciesLearnset: LearnsetData | undefined = Utils.deepClone(mod.species.getLearnset(usedSpecies.id));
```

Comment:
Realistically couldn't the typecast here just be removed?

Reply:
Yes, but it resulted in a somewhat messy type:
```
Mutable<{
    [moveid: string]: string[];
} | undefined>
```

But I def prefer leaving the type off, so I'll update it